sunkit_image/tests/test_trace.py

```diff
@@ -0,0 +1,155 @@
+import matplotlib.pyplot as plt
+import astropy
+import numpy as np
+import pytest
+import os
+
+import sunkit_image.trace as trace
+import sunkit_image.data.test as data
+from sunpy.tests.helpers import figure_test
+
+
+@pytest.fixture
+@pytest.mark.remote_data
+def image():
+
+    im = astropy.io.fits.getdata("http://www.lmsal.com/~aschwand/software/tracing/TRACE_19980519.fits", ignore_missing_end=True)
+    return im
+
+
+@pytest.fixture
+def filepath_IDL():
+
+    filepath = data.get_test_filepath("IDL.txt")
+    return filepath
+
+
+@pytest.fixture
+def filepath_py():
+
+    filepath = data.get_test_filepath("Python.txt")
+    return filepath
+
+
+@pytest.mark.remote_data
+def test_occult2_remote(image, filepath_IDL, filepath_py):
+
+    # Testing on the same input files as in the IDL tutorial
+    loops = trace.occult2(image, nsm1=3, rmin=30, lmin=25, nstruc=1000, ngap=0, qthresh1=0.0, qthresh2=3.0, file=True)
+
+    result = np.loadtxt("loops.txt")
+
+    os.remove("loops.txt")
+
+    # First we will verify on the Python version of the output which we know is correct as we had plotted
+    # it earlirer and also have made visual comparisons
+    expect = np.loadtxt(filepath_py)
```

Comment:
Use https://pypi.org/project/pytest-arraydiff/ to do these tests instead of opening the file.